docs/source/quantization.rst

```diff
@@ -193,7 +193,7 @@ Basic activations are supported.
 * :meth:`~torch.nn.functional.adaptive_avg_pool2d` - Adaptive average pooling
 * :meth:`~torch.nn.functional.avg_pool2d` - Average pooling
 * :meth:`~torch.nn.functional.interpolate` - Interpolation
-* :meth:`~torch.nn.functional.hardswish` - Hard Swish
+* :meth:`~torch.nn.functional.hardswish` - Hardswish
```

Comment:
actually why are these method? I feel it makes more sense for these to be `func`

Reply:
not sure if there is any difference in the generated docs.  If this is important, might be good to be a separate PR on the entire file, so that it's consistent within the file

Reply:
yeah, it should be changed in a separate PR I think